src/main/java/org/fogbowcloud/manager/core/ManagerController.java

```diff
@@ -41,4 +52,16 @@ private void startManagerThreads() {
 		this.openProcessorThread.start();
 	}
 
-}
\ No newline at end of file
+	public void setComputePlugin(ComputePlugin computePlugin) {
```

Comment:
@italo-batista not sure what is the better ... can we pass these object via the constructor instead o setMethods?

Reply:
I wanted to avoid that the ManagerController's constructor was too big, since there will be a lot of plugins. But I recognize your suggestion is safer. I will change it.